script/github-actions/get-changed-apps.unit.spec.js

```diff
@@ -13,6 +13,10 @@ describe('getChangedAppsString', () => {
     });
   };
 
+  const createChangedAppsConfig = (apps = []) => {
```

Comment:
Even though this isn't doing much, I added this just in case we change the structure again in the future